klippa/src/parsing_util.rs

```diff
@@ -86,6 +87,48 @@ pub fn parse_drop_tables(input_str: &str) -> Result<IntSet<Tag>, SubsetError> {
     Ok(result)
 }
 
+//parse input name_IDs string, which is a comma/whitespace-separated list of nameIDs that will be retained
+pub fn parse_name_ids(input_str: &str) -> Result<IntSet<NameId>, SubsetError> {
+    let mut result = IntSet::empty();
+
+    if input_str == "*" {
+        let out = IntSet::<NameId>::all();
+        return Ok(out);
+    }
+
+    for name_id in input_str.split(|c| c == ',' || c == ' ') {
+        if name_id.is_empty() {
+            continue;
+        }
+        let name_id = name_id
+            .parse::<u16>()
+            .map_err(|_| SubsetError::InvalidId(name_id.to_owned()))?;
+        result.insert(NameId::from(name_id));
+    }
+    Ok(result)
+}
+
+//parse input name_languages string, which is a comma/whitespace-separated list of langIDs that will be retained
+pub fn parse_name_languages(input_str: &str) -> Result<IntSet<u16>, SubsetError> {
+    let mut result = IntSet::empty();
+
+    if input_str == "*" {
+        let out = IntSet::<u16>::all();
+        return Ok(out);
+    }
+
+    for lang_id in input_str.split(|c| c == ',' || c == ' ') {
+        if lang_id.is_empty() {
+            continue;
+        }
+        let lang_id = lang_id
+            .parse::<u16>()
+            .map_err(|_| SubsetError::InvalidId(lang_id.to_owned()))?;
+        result.insert(lang_id);
+    }
+    Ok(result)
+}
```

Comment:
Same feedback as above. Also parse_name_ids and parse_name_languages are basically identical, we should be able to exploit that.